diff --git a/dist/index.js b/dist/index.js
index 9bc5ed0c0ebceded126ca436835bbf2042864c28..59bcb219d61645457b59884e0768ee51bf9659e3 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,7 +1,7 @@
 import { visit } from 'unist-util-visit';
 import { toString } from 'hast-util-to-string';
 import rangeParser2 from 'parse-numeric-range';
-import { getHighlighter } from 'shikiji';
+import { getHighlighterCore as getHighlighter } from 'shikiji/core';
 import { unified } from 'unified';
 import rehypeParse from 'rehype-parse';
 
