diff --git a/dist/index.js b/dist/index.js
index 4689748d448476fd1ae35ba475cb20feefe426a7..0a8a9eb197cecf84f3340e73e4db186c5a98490c 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,7 +1,7 @@
 import { visit } from 'unist-util-visit';
 import { toString } from 'hast-util-to-string';
 import rangeParser2 from 'parse-numeric-range';
-import { getHighlighter } from 'shikiji';
+import { getHighlighterCore as getHighlighter } from 'shikiji/core';
 import { unified } from 'unified';
 import rehypeParse from 'rehype-parse';
 
